packages/create-strapi-app/utils/prompt-user.js

```diff
@@ -0,0 +1,93 @@
+'use strict';
+
+const inquirer = require('inquirer');
+const axios = require('axios');
+const yaml = require('js-yaml');
+const logger = require('../../create-strapi-starter/utils/logger');
+
+/**
+ * @param {Object} projectArgs - The arguments needed to create a project
+ * @param {string|null} projectArgs.projectName - The name/path of project
+ * @param {string|null} projectArgs.template - The Github repo of the template
+ * @param {boolean} projectArgs.useQuickstart - Check quickstart flag was set
+ * @param {string} cliType - The type of cli: starters or templates
+ * @returns
+ */
+module.exports = async function promptUser(projectArgs) {
+  const questions = await getPromptQuestions(projectArgs);
+  const prompt = await inquirer.prompt(questions);
+
+  return prompt;
+};
+
+async function getPromptOptions() {
```

Comment:
this is only for template not for all options you might want to rename it to something more meaningful